docs/EXTERNAL_PLUGINS.md

```diff
@@ -39,7 +39,7 @@ This is a guide to help you set up your plugin to use it with `execd`
   in the new repo. For a further example of what this might look like, take a
   look at [ssoroka/rand](https://github.com/ssoroka/rand) or
   [danielnelson/telegraf-execd-openvpn](https://github.com/danielnelson//telegraf-execd-openvpn)
-  1. Copy [main.go](./example/cmd/main.go) into your project under the `cmd` folder.
+  1. Copy [main.go](/plugins/inputs/execd/shim/example/cmd/main.go) into your project under the `cmd` folder.
```

Comment:
```suggestion
  1. Copy [main.go](/plugins/common/shim/example/cmd/main.go) into your project under the `cmd` folder.
```

Reply:
The inputs version of this should be avoided, and will probably be redirected at the common/shim version.